oo-install/atomic-openshift-utils.spec

```diff
@@ -0,0 +1,58 @@
+%{!?python_sitelib: %global python_sitelib %(%{__python} -c "from distutils.sysconfig import get_python_lib; print get_python_lib()")}
+
+%global import_path github.com/openshift/openshift-extras
+# %commit is intended to be set by tito custom builders provided
+# in the .tito/lib directory. The values in this spec file will not be kept up to date.
+%{!?commit:
+%global commit c64d09e528ca433832c6b6e6f5c7734a9cc8ee6f
+}
+
+Name:           atomic-openshift-utils
+Version:        3.0.0
+Release:        1%{?dist}
+Summary:        Atomic OpenShift Utilities
+License:        ASL 2.0
+URL:            http://github.com/openshift/openshift-extras/tree/enterprise-3.0/oo-install
+Source0:        https://%{import_path}/archive/%{commit}/%{name}-%{version}.tar.gz
+BuildArch:      noarch
+
+BuildRequires: python-setuptools
+Requires:      ansible
+Requires:      python-click
+Requires:      python-ecdsa
+Requires:      python-jinja2
+Requires:      python-markupsafe
+Requires:      python-paramiko
+Requires:      python-crypto
```

Comment:
Hmm, is python-crypto used in oo-install? Suspect that's actually an ansible dep, we may not need it here. Looking at the docs for it all modules appear to be namespaced under Crypto, and there are no hits for that in oo-install.

Actually I think this may also be true for python-ecdsa, python-jinja2, python-markupsafe, and python-paramiko, I think these were all ansible deps which would be carried by our dep on ansible itself now.